contributors/yash-raj-bharti/index.md

```diff
@@ -0,0 +1,37 @@
+---
+contributor_name: Yash Raj Bharti
+folder_name: yash-raj-bharti
+is_featured: true
+img_alt: A person standing next to a colorful graffiti-painted Android statue, with the Google headquarters in the background. The person is wearing a lanyard on a sunny day.
+usernames:
+  github: yashrajbharti
+quote: MDN resonates with my passion to build a consistent and open web, where developers can learn and grow.
+---
+
+Hi! I'm' Yash, a UXE and a Senior Mentor of the Liquid Galaxy, where I work on GIS applications, making immersive data viz projects on Google Maps, Streetview, and Earth. Liquid Galaxy is a Google-founded open-source project and was my primary reason for falling in love with open-source. I work on the web: HTML, CSS, and JS, and I believe that honing strong basics can make even the most complex tasks easy.
+
+I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg(a contributor) at the time meant a lot. It was for a [PWA](https://github.com/mdn/content/pull/29488), and I said: iTunes value associated with the App Store for platform members of related apps has been dropped since it's misleading and currently not supported by Safari. The list has been updated as per the W3C wiki.
+
+Since then, I have been lurking to find a way to contribute to MDN, I also contributed to web.dev to learn more. I am not new to MDN W3C, or Web.dev (HTML5Rocks), and have been learning like a nerd for the last 9 years, I was a couple of years back I started contributing, felt like someone helping the space, making explainers, worked alongside people like Adam Argyle from [CSS motion blur](https://github.com/w3c/csswg-drafts/issues/11134) and Thomas Steiner(Project Fugu). I discovered, that when I put my passion and heart into it, I can achieve, the grit to solve challenges isn't new to me.
```

Comment:
```suggestion
Since then, I have been lurking to find a way to contribute to MDN. I also contributed to [web.dev](https://web.dev/) to learn more. I am not new to MDN, W3C, or web.dev (HTML5Rocks), and have been learning like a nerd for the last 9 years, but a couple of years back, I started finally contributing. I felt like helping the community, making explainers, and I worked alongside people like Adam Argyle on a [CSS motion blur proposal](https://github.com/w3c/csswg-drafts/issues/11134) and Thomas Steiner ([Project Fugu](https://www.chromium.org/teams/web-capabilities-fugu/)). I discovered, that when I put my passion and heart into something, I can achieve it. The grit to solve challenges isn't new to me.
```